pkg/machine/e2e/basic_test.go

```diff
@@ -84,8 +83,8 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(stopSession).To(Exit(0))
 
 		// gxproxy should exit after machine is stopped
-		_, err = exec.Command("pgrep", "gvproxy").Output()
-		Expect(err).To(HaveOccurred())
+		out, _ = pgrep("gvproxy")
+		Expect(out).To(ContainSubstring("gvproxy"))
```

Comment:
Sorry my bad, I assume you need `ToNot()` here as you check for not running right?